app/assets/src/loader.scss

```diff
@@ -55,5 +56,6 @@
 @import "styles/views/taxon_tree_vis";
 
 // visualizations
+@import "styles/visualizations/heatmap/heatmap";
```

Comment:
Would be great to not add more `.scss` files to this file. If you use CSS Modules, you don't need to import the file here any more.

Reply:
this is for the d3 file. And i ran into issues applying to d3. Will fix in a follow up.